hazelcast/src/test/java/com/hazelcast/map/impl/eviction/ExpirationManagerTest.java

```diff
@@ -37,7 +37,7 @@
 import static java.lang.System.setProperty;
 import static org.junit.Assert.assertEquals;
 
-@RunWith(HazelcastParallelClassRunner.class)
+@RunWith(HazelcastSerialClassRunner.class)
```

Comment:
Why change this?

Reply:
See commit message here
https://github.com/hazelcast/hazelcast/pull/12262/commits/b671cc548cb5a756c74c4c3441e2938484e1a35a